### ISSUE TITLE
g:started_by_firenvim is not set on windows

### ISSUE THREAD

**Issue by BlueDrink9** [glacambre/firenvim#379]
Sorry, me again.

- OS Version: win x86-64
- Browser Version: Google Inc. Chrome 79.0.3945.130
- Browser Addon Version: 0.1.17
- Neovim Plugin Version: 0.1.17
- What I tried to do: Change config when started by firenvim
- What happened: No config changed, `echom g:started_by_firenvim` gives undefined variable error.

**Comment by glacambre** [comment]
Duplicate of https://github.com/glacambre/firenvim/issues/315 . Try updating the neovim plugin and running `firenvim#install` again :)

**Comment by BlueDrink9** [comment]
Oops sorry. Thanks, that worked

**Comment by BlueDrink9** [comment]
Hmm, doesn't seem to have worked, even after updating.

Ah, but I see you haven't released the relevant commit yet. I'll wait on that.

Are releases automatic? (Done on CI?) If not, would you be interested in having it set up?

**Comment by glacambre** [comment]
>Ah, but I see you haven't released the relevant commit yet. I'll wait on that.

This shouldn't matter for the neovim plugin. Could you run `firenvim#uninstall` and `firenvim#install` again? Perhaps, while your browser is closed? If this still doesn't work, could you upload the content of the [firenvim script](https://github.com/glacambre/firenvim/blob/master/TROUBLESHOOTING.md#make-sure-the-firenvim-script-has-been-created) here?

>Are releases automatic? 

No, they aren't, they require manually uploading files to the chrome & firefox stores. I haven't checked if there were APIs to do that though.

**Comment by BlueDrink9** [comment]
To clarify, I'm only pulling tags, to make sure my firenvim version is never ahead of release versions. If you want I can manually pull the new one, or if you think there are no downsides I'll change back to pulling from master.

**Comment by glacambre** [comment]
Only pulling tags isn't a bad idea (there shouldn't be any issues with using master but I don't pay an enormous amount of attention to compatibility, so it's better to be safe than sorry :)). I released v0.1.18 a few hours ago, it should reach the chrome web store in a few days, let's wait until that happens and see if the issue persists :)

**Comment by BlueDrink9** [comment]
Thank you!

**Comment by BlueDrink9** [comment]
Old article, but looks like CI deployment is possible for [chrome](https://circleci.com/blog/why-continuous-integration-is-key-to-stepping-up-deployment-frequency/). [Firefox should be possible too](https://alosarjos.gitlab.io/alosarblog/post/continuous-integration-for-a-firefox-extension/), but may have issues with signing. ([These are being addressed](https://github.com/mozilla/web-ext/issues/804).)
